src/builtins.ts

```diff
@@ -9421,6 +9424,18 @@ function builtin_f64x2_ge(ctx: BuiltinContext): ExpressionRef {
 }
 builtins.set(BuiltinNames.f64x2_ge, builtin_f64x2_ge);
 
+// access arguments.length
+function builtin_arguments_length(ctx: BuiltinContext): ExpressionRef {
+  const compiler = ctx.compiler;
+  const mod = compiler.module;
+  compiler.currentType = Type.i32;
+  const local = compiler.currentFlow.lookupLocal(BuiltinNames.argumentsLengthLocal)!;
+  return mod.local_get(local.index, TypeRef.I32);
```

Comment:
This becomes the function body of the builtin. Using `arguments.length` accesses the namespace, and calls the function. This is not desired behavior.

I believe we need to override `compilePropertyAccess()`